core_debug.tex

```diff
@@ -42,10 +42,14 @@ \section{Single Step}
 A debugger can cause a halted hart to execute a single instruction and then
 re-enter Debug Mode by setting \Fstep before setting \Fresumereq.
 
-If the instruction that is executed causes an exception, Debug Mode is re-entered
-immediately after the PC is changed to the exception handler and and the
+If executing or fetching that instruction causes an exception, Debug Mode is
+re-entered immediately after the PC is changed to the exception handler and the
 appropriate {\tt badaddr} and {\tt cause} registers are updated also.
```

Comment:
this looks good, but drop the last 'also'?

Reply:
Yes.